src/hapi/pipelines/database/org.py

```diff
@@ -1,21 +1,37 @@
 import logging
+from typing import Dict
 
 from hapi_schema.db_org import DBOrg
+from hdx.location.names import clean_name
+from hdx.scraper.utilities.reader import Read
 from sqlalchemy import select
 from sqlalchemy.orm import Session
 
-from hapi.pipelines.database.base_uploader import BaseUploader
+from .base_uploader import BaseUploader
 
 logger = logging.getLogger(__name__)
 
 
 class Org(BaseUploader):
-    def __init__(self, session: Session):
+    def __init__(
+        self,
+        session: Session,
+        datasetinfo: Dict[str, str],
+    ):
         super().__init__(session)
+        self._datasetinfo = datasetinfo
         self.data = {}
+        self._org_map = {}
 
     def populate(self):
-        raise NotImplementedError()
+        logger.info("Populating org mapping")
+        reader = Read.get_reader()
+        headers, iterator = reader.get_tabular_rows(
```

Comment:
Another thing to note is the file names of the mapping tables (org, sector, org_type).  They seem to get the names of the scraper that was last run, so they're named "food_security_ch_1sxsircxhxmzce0pwafcht-whjbdi9uqoefesubilgc4-export-format-csv-gid-2037574879.csv".  Should I override the default names in get_tabular_rows?